### ISSUE TITLE
4.2 breaks on build: #if targetEnvironment(simulator)

### ISSUE THREAD

**Issue by bbaic** [ashleymills/Reachability.swift#309]
Installed via cocoapods v1.5.3
Xcode 9.2
deployment target: 11.0

v 4.1.0 works fine; 4.2.0 won't build (see attached)

Reachability.swift line 105 and 248:
![screen shot 2018-09-07 at 10 38 38 am](https://user-images.githubusercontent.com/1171334/45228976-448b2200-b28a-11e8-9fbe-2bc5a03fb66f.png)

https://github.com/ashleymills/Reachability.swift/issues/285 was meant to resolve a warning in xcode 9.3, but the implemented change breaks things in 9.2; a warning in 9.3 is far better than broken in 9.2...

**Comment by hipwelljo** [comment]
You can use an old Reachability release if you're using an old version of Xcode.

**Comment by bbaic** [comment]
True, and for now I will. At the very least this needs to be documented though.

**Comment by bbaic** [comment]
@ashleymills works for me — appreciate your efforts developing and maintaining this!
